recipes/lessR/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set version = '3.9.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-lessr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/lessR_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/lessR/lessR_{{ version }}.tar.gz
+  sha256: 1f787748b2ddd4d0ce411651869f34adf9b2e91463cd4be3b9c03b9a53072aea
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{posix}}zip               # [win]
+
+  host:
+    - r-base
+    - r-colorspace
+    - r-ellipse
+    - r-foreign
+    - r-lattice
+    - r-latticeextra
+    - r-leaps
+    - r-openxlsx
+    - r-png
+    - r-rmarkdown
+    - r-robustbase
+    - r-sas7bdat
+    - r-triangle
+    - r-viridislite
+    - r-wesanderson
+
+  run:
+    - r-base
+    - r-colorspace
+    - r-ellipse
+    - r-foreign
+    - r-lattice
+    - r-latticeextra
+    - r-leaps
+    - r-openxlsx
+    - r-png
+    - r-rmarkdown
+    - r-robustbase
+    - r-sas7bdat
+    - r-triangle
+    - r-viridislite
+    - r-wesanderson
+
+test:
+  commands:
+    - $R -e "library('lessR')"           # [not win]
+    - "\"%R%\" -e \"library('lessR')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=lessR
+  license: GPL-2
+  summary: Each function accomplishes the work of several or more standard R functions. For example,
+    two function calls, Read() and CountAll(), read the data and generate summary statistics
+    for all variables in the data frame, plus histograms and bar charts as appropriate.  Other
+    functions provide for descriptive statistics, a comprehensive regression analysis,
+    analysis of variance and t-test, plotting including the introduced here Violin/Box/Scatter
+    plot for a numerical variable, bar chart, histogram, box plot, density curves, calibrated
+    power curve, reading multiple data formats with the same function call, variable
+    labels, color themes, Trellis graphics and a built-in help system. Also includes
+    a confirmatory factor analysis of multiple indicator measurement models, pedagogical
+    routines for data simulation such as for the Central Limit Theorem, and generation
+    and rendering of R markdown instructions for interpretative output.
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
```

Comment:
Can this be taken from the source using the `{{ SRC_DIR }}` variable instead
https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html